### ISSUE TITLE
"Login failed after passphrase verified" in macOS client

### ISSUE THREAD

**Issue by ssemenova** [keybase/client#15720]
Logging into the mac client, I get the error "Login failed after passphrase verified". This is a distinctly different error than the error I get if typing in an incorrect passphrase.

I can log into the web client with the same passphrase with no errors. My PGP key is stored with keybase, and using the web client, I can successfully encrypt and decrypt messages with other users.

I tried, to no avail:
1. Changing my password through the keybase web client. This successfully changed my password, but the new password has the same error.
2. Uninstalling keybase by first running "keybase uninstall", then deleting the keybase app, then re-installing.

my log id, by just running "keybase log send": 564177cfa2822ea3c2d2a91c
my log id, by running "keybase log send" immediately after attempting to log in: 361ccee1fa4171128a338d1c

**Comment by maxtaco** [comment]
Cc @zapu did we recently have this bug?

@ssemenova I think it is because your account was established a very long time ago and we made some changes that broke this vintage.

**Comment by eblanton** [comment]
I assume you can see this, but @ssemenova's account was created 2018-03-25, which is not exactly forever ago.  Mine is 2014-03-14, and my account is working OK.

**Comment by maxtaco** [comment]
Ah thanks. I was tapping on my phone so didn’t see that.

**Comment by maxtaco** [comment]
@eblanton do you and @ssemenova share a computer?

**Comment by eblanton** [comment]
Negative.  We simply work together.

**Comment by maxtaco** [comment]
@seojangho: can you maybe try this from the CLI:

```
keybase logout
keybase login novasofiya
```

**Comment by ssemenova** [comment]
@maxtaco No. I did get a new computer a few months ago (I think this is my first time logging out of it) and had to add it as a new device to my account, but I'm not sure that would cause this.

**Comment by maxtaco** [comment]
... I'm guessing that's not going to work... This is somewhat of a surprise, but it looks as if your secret key file has been corrupted..  It might be necessary to revoke the device and then reprovision it again. You won't lose any data, but it's annoying...

**Comment by ssemenova** [comment]
@maxtaco Yeah, your guess was right, it didn't work. I have no idea how the file could have become corrupted but I can revoke the device and try to reprovision.

**Comment by ssemenova** [comment]
@maxtaco Yep, that worked. Thank you!
